### ISSUE TITLE
Move away from github pages

### ISSUE THREAD

**Issue by juliendonck** [radicle-dev/radicle.xyz#274]
As some people mentioned on discord, it doesn't make sense for our website to be hosted on github pages since when github is down, people will want to look for alternatives, they will find us but our website will be unavailable.

**Comment by juliendonck** [comment]
@cloudhead do you think this makes sense and if so, do you have an idea for alternative set ups?

**Comment by cloudhead** [comment]
Yeah, sure. We use netlify for radicle.foundation, we could use that, or fleek.
